client/faucet/src/mint.rs

```diff
@@ -147,18 +158,19 @@ impl Service {
             if let (Some(ref diem_id_domain), Some(is_remove_domain)) =
                 (&params.diem_id_domain, params.is_remove_domain)
             {
-                let builder = if is_remove_domain {
-                    self.transaction_factory.remove_diem_id_domain(
+                if is_remove_domain {
```

Comment:
should we ignore remove when the domain is not inside the map? because otherwise we probably get an error, and causes client to retry.